llarp/config/config.cpp

```diff
@@ -175,23 +194,37 @@ namespace llarp
     });
 
     conf.defineOption<int>("network", "paths", false, PathsDefault, [this](int arg) {
-      if (arg < 1 or arg > 8)
-        throw std::invalid_argument("[endpoint]:paths must be >= 1 and <= 8");
+      if (arg < 2 or arg > 8)
+        throw std::invalid_argument("[endpoint]:paths must be >= 2 and <= 8");
       m_Paths = arg;
     });
 
     conf.defineOption<std::string>("network", "exit-node", false, "", [this](std::string arg) {
       if (arg.empty())
         return;
       service::Address exit;
+      IPRange range;
+      const auto pos = arg.find(":");
+      if (pos == std::string::npos)
+      {
+        range.FromString("0.0.0.0/0");
+      }
+      else if (not range.FromString(arg.substr(pos + 1)))
+      {
+        throw std::invalid_argument("[network]:exit-node invalid ip range for exit provided");
+      }
```

Comment:
So this lets me write something like:

```
exit-node=abc.loki:8.0.0.0/8
exit-node=def.loki:14.2.0.0/16
```
but a couple questions:
1. What happens if the ranges overlap, either partially such as:
```
exit-node=abc.loki:8.0.0.0/8
exit-node=def.loki   # (implied :0.0.0.0/0 default)
```
or entirely:
```
exit-node=abc.loki:8.0.0.0/8
exit-node=def.loki:8.0.0.0/8
```
2. Does this support using the same exit multiple times, so that I can map multiple subnets to an exit?
```
exit-node=abc.loki:8.0.0.0/8
exit-node=abc.loki:123.0.0.0/8
```

(This could use some comment, at least in the code, explaining how this is supposed to work, which exits get priority, etc.)